radis/lbl/factory.py

```diff
@@ -1182,8 +1181,11 @@ def eq_spectrum_gpu(
             Tgas,
             mole_fraction,
             verbose=verbose,
-            gpu=(not emulate),
         )
+
+        if exit_gpu:
+            gpu_exit()
```

Comment:
Do you need try/except/finally and add gpu_exit() in finally?

Reply:
It is inside the if statement because if eq_spectrum_gpu() is called from within the interactive plots, gpu_exit() shouldn't be called in eq_spectrum_cpu() because there will still follow gpu_iterate() calls. This is controlled by passing False when called from the interactive_gpu

Reply:
OK! Maybe add this comment directly in the code under the `if` statement?

Reply:
Done!